lib/partitions_validator_utils.pm

```diff
@@ -23,7 +23,6 @@ our @EXPORT = qw(
 
 sub validate_partition_table {
     my $args = shift;
-    return if check_var('BACKEND', 's390x');    # blkid output does not show partition table for dasd
```

Comment:
Is there a workaround now ? or we do not use this at all on zVM ?

Reply:
I got confused because I forgot about zVM and thought that this will never run so I removed it, I will change it.